.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: php
 php:
-    - 5.3
```

Comment:
Please let the world use DictionaryBundle with php 5.3 :crying_cat_face: 
(yes, the whole world is using this bundle)